### PR TITLE
fix theme generator rendering

### PR DESCRIPTION
**Pull request by apoorvkhare07**
fix theme generator rendering

### Description

This fix the rendering issue with the current theme-generator, by converting the css to scss files and make other required changes. 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by shucon** [comment]
Travis-CI has failed

File: themes/pmahomme/scss/theme_generator.scss [hunk 1]
```diff
@@ -0,0 +1,709 @@
...
+#group_preview.group {
+    width: 294px;
+}
+
+#table_preview {
+    display: none;
+    position: relative;
+    left: 523px;
+    bottom: 718px;
+}
```
Comment (shucon) [inline themes/pmahomme/scss/theme_generator.scss]: File should end with a new line.

**Comment by shucon** [comment]
@sudo-bot Deploy PR

**Comment by williamdes** [comment]
@shucon I added you to the authorized list of users ;)

**Comment by shucon** [comment]
@williamdes Thanks, I was wondering why it didn't work :stuck_out_tongue:

**Comment by shucon** [comment]
@williamdes it's facing some problems to deploy, is it because this branch is behind master branch?

**Comment by williamdes** [comment]
@shucon It seems to be finished now ?

**Comment by shucon** [comment]
@williamdes are you able to open the url?

**Comment by williamdes** [comment]
@shucon Yes, please try to use other DNS servers ;)
`1.1.1.1` and `1.0.0.1` https://1.1.1.1

image

**Comment by apoorvkhare07** [comment]
> Travis-CI has failed

That's probably because the branch is many commits behind master.

**Comment by williamdes** [comment]
> That's probably because the branch is many commits behind master.

cc @mauriciofauth

**Comment by apoorvkhare07** [comment]
@williamdes @shucon I can make my changes in current master and then push to this branch if we want to make this branch in sync with the master. But there will be some merge conflicts in that case i suppose.

**Comment by mauriciofauth** [comment]
@apoorvkhare07 Did you created a branch from `master` or `gsoc-theme-generator`?

**Comment by apoorvkhare07** [comment]
@mauriciofauth `master`

**Comment by shucon** [comment]
@apoorvkhare07 The branch should be created from `gsoc-theme-generator`.

**Pull request closed without merging**